sklearn/linear_model/least_angle.py

```diff
@@ -1479,7 +1479,7 @@ def __init__(self, criterion='aic', fit_intercept=True, verbose=False,
         self.eps = eps
         self.fit_path = True
 
-    def fit(self, X, y, copy_X=True):
+    def fit(self, X, y, copy_X=None):
```

Comment:
docstring should be updated when you change a function signature.

you'll have to update what's new page too.

Reply:
noted, thanks. as soon as we decide on the deprecation I'll do it (since the deprecation guidelines require using something else than None)

Reply:
for me it's a bug fix and not an API change. So I would be fine as done here.

Reply:
got it, thanks. Is there a way to edit what's new page besides manually editing html and submitting pull request on scikit-learn.github.io? I cannot find any references in the contribuiting or developer's guidelines .